### ISSUE TITLE
Unable to install on EdgeMax router

### ISSUE THREAD

**Issue by kubat** [nextdns/nextdns#250]
-  Following instructions from [here](https://github.com/nextdns/nextdns/wiki/EdgeOS).
  - The shell command fails silently.
  - Running just the `curl` portion without `-s` flag gives the following error:

```console
ubnt@ubnt:~$ curl -vL https://nextdns.io/install
*   Trying 96.114.156.37...
* TCP_NODELAY set
* Connected to nextdns.io (96.114.156.37) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs
* TLSv1.2 (OUT), TLS header, Certificate Status (22):
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (OUT), TLS alert, Server hello (2):
* SSL certificate problem: certificate has expired
* Curl_http_done: called premature == 1
* stopped the pause stream!
* Closing connection 0
curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
```

  - I do not have curl issues when attempting on my laptop (OS X).
  - Execution of the install script (`scp`'ed to the router) results in the following output:

```console
ubnt@ubnt:~$ sh -c 'sh -c "$(cat nextdns.io.install.sh)"'
INFO: OS: edgeos
INFO: GOARCH: mips64_softfloat
INFO: GOOS: linux
INFO: NEXTDNS_BIN: /config/nextdns/nextdns
INFO: LATEST_RELEASE: 
ERROR: Cannot detect running environment.
```

**Context**
 - Version [e.g. 1.5.2]: 
 - Platform [e.g. macOS, ASUS-Merlin]: EdgeOS v2.0.8-hotfix.1 (current latest)
 - `curl` version info:

```console
ubnt@ubnt:~$ curl -V
curl 7.52.1 (mips-unknown-linux-gnu) libcurl/7.52.1 OpenSSL/1.0.2u zlib/1.2.8 libidn2/0.16 libpsl/0.17.0 (+libidn2/0.16) libssh2/1.7.0 nghttp2/1.18.1 librtmp/2.3
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz TLS-SRP HTTP2 UnixSockets HTTPS-proxy PSL 
```

**Comment by rs** [comment]
Please check the time is correctly synced on your router.

**Comment by rnicholus** [comment]
I has the same problem, and I resolved it by doing two things:

1. Fixing the system time on my ERX (it was strangely off by 6 hours - not sure how that suddenly happened).
2. Fixing the system name-server. My system DNS was oddly pointed to a disabled local network. Not sure how that happened either. 

Once I fixed those two items, i reinstalled nextdns on my ERX, and all was well.

All of these issues appears after upgrading to EdgeOS 2.0.8-hotfix.1. Downgrading did not fix by itself.
